config/agencies.yml

```diff
@@ -0,0 +1,23 @@
+test:
```

Comment:
It seems like this could just be an array of agency names, without the `id` as the key. With that you could use the existing agency name in `service_providers.yml` instead of a number which I think would read a little bit nicer.

```yml
test:
  - CBP
  - OPM
  - EOP
```

Reply:
It also looks like we're missing agencies. For example, we should probably add GSA to the list for all of our apps, right?

Reply:
I wanted to leave the agency names in service providers alone since it is loosely related to the agencies we are creating (which are really agency groupings).  Eventually we could cleanup those non-normalized names in service providers.  I chose to use ids in agencies since it made the code and the migration a lot simpler.  Eventually we want to have all the sp's have an agency but rather than get that definitive list I chose to have the code default gracefully and allow us to add new agencies as needed.  I should probably hit the group with the list and have everybody decide what agencies they want associated with the sp's that haven't been assigned.